### PR TITLE
shows all exercises for test cohort no matter the language

### PR DESCRIPTION
**Pull request by merleschoen**
This pr allows the cohort language to be null, so teachers can see all exercises, regardless of the student's chosen language. It also includes a sql query to update the language of the cohort testing the learning cycle implementation to null.

### CODE REVIEW

**Review by mircealungu — APPROVED** [review]
looks good.

File: tools/migrations/24-05-06--update_cohort_language_to_null_for_testing.sql [hunk 1]
```diff
@@ -0,0 +1,4 @@
+/* 
+ Sets language id for learning cycle test cohort to null, so we can see all exercise despite the language.
+ */
+UPDATE cohort SET language_id = NULL WHERE name IN ('Merle', 'MerleITU');
```
Comment (mircealungu) [inline tools/migrations/24-05-06--update_cohort_language_to_null_for_testing.sql:4]: ah, this has to be `invite_code` not `name`. otherwise good!
